Loop/Managers/LoopDataManager.swift

```diff
@@ -147,11 +223,37 @@ class LoopDataManager {
             }
         }
 
+        // Not used directly in loop decision making, but used for reporting loop state
```

Comment:
Hmm. `WatchDataManager` and `StatusTabkeViewController` are both clients of Loop data but neither requires embedding in the LDM itself. Can I suggest that approach?